server/server.go

```diff
@@ -376,8 +376,19 @@ func New(
 	if s.appArmorEnabled && config.ApparmorProfile == libconfig.DefaultApparmorProfile {
 		logrus.Infof("installing default apparmor profile: %v", libconfig.DefaultApparmorProfile)
 		if err := apparmor.InstallDefault(libconfig.DefaultApparmorProfile); err != nil {
-			return nil, fmt.Errorf("ensuring the default apparmor profile %q is installed failed: %v", libconfig.DefaultApparmorProfile, err)
+			return nil, errors.Wrapf(err,
+				"installing default apparmor profile %q failed",
+				libconfig.DefaultApparmorProfile,
+			)
 		}
+		profileContent, err := apparmor.DefaultContent(libconfig.DefaultApparmorProfile)
```

Comment:
Can we make this conditional to only execute it when needed?
```golang
if logrus.IsLevelEnabled(logrus.TraceLevel) {
   profileContent, err := apparmor.DefaultContent(...
}
```

Reply:
Yes, I changed it as suggested.